scifin/classifier/classifier.py

```diff
@@ -73,6 +73,8 @@ def dtw_distance(ts1, ts2, window=None):
       Second time series.
     window : int
       Size of restrictive search window.
+    mode : str
+      Mode to choose from ('abs' or 'square').
```

Comment:
add some words explaining what 'abs' and 'square' mean here?

Reply:
Indeed. Thanks.